core/codegen/src/lib.rs

```diff
@@ -114,14 +113,14 @@ mod syn_ext;
 
 use crate::http::Method;
 use proc_macro::TokenStream;
-crate use devise::proc_macro2;
+pub(crate) use devise::proc_macro2;
 
```

Comment:
Since it's in a crate root, this should have the same effect as no visibility modifier at all as access is always granted to children. If that is the case, let's remove the `pub(crate)` here to stay consistent with other `use`s.

Reply:
Good catch. There's this in a couple places with `mod` instead of `use` as well.